### ISSUE TITLE
`no-unused-svelte-ignore` parses `svelte-ignore` comments differently than `svelte` 5 does

### ISSUE THREAD

**Issue by ArtskydJ** [sveltejs/eslint-plugin-svelte#1449]
Svelte 5 is cool with this `svelte-ignore` comment
```
// svelte-ignore name_of_warning (whatever text i want)
```
but this plugin, using the `no-unused-svelte-ignore` rule treats "(whatever text i want)" as if each word is a name of a rule.  It thinks "(whatever" is an unused ignored rule, same for "text", and "i".

# Replication

https://svelte.dev/playground/7d1bf802635d4b6aa4e7e19f2e616090?version=5.46.0

```svelte
<script>
	let name = $state('world');

	// svelte-ignore state_referenced_locally (this is in parens and should be ignored)
	let initial_name = name
</script>
...
```

According to [Svelte docs](https://svelte.dev/docs/svelte/compiler-warnings)
> You can list multiple rules in a single comment (separated by commas), and add an explanatory note (in parentheses) alongside them:
> ```svelte
> 
> <div onclick>...</div>
> ```

## Actual behavior

And if you run eslint with eslint-plugin-svelte with no-unused-svelte-ignore, then you'll get errors like this:
```
   4:44  error  svelte-ignore comment is used, but not warned  svelte/no-unused-svelte-ignore
   4:50  error  svelte-ignore comment is used, but not warned  svelte/no-unused-svelte-ignore
   4:53  error  svelte-ignore comment is used, but not warned  svelte/no-unused-svelte-ignore
   4:56  error  svelte-ignore comment is used, but not warned  svelte/no-unused-svelte-ignore
   4:63  error  svelte-ignore comment is used, but not warned  svelte/no-unused-svelte-ignore
   4:67  error  svelte-ignore comment is used, but not warned  svelte/no-unused-svelte-ignore
   4:74  error  svelte-ignore comment is used, but not warned  svelte/no-unused-svelte-ignore
```
and if I line up the columns with the comment:
```
	// svelte-ignore state_referenced_locally (this is in parens and should be ignored)
	//                                        ^     ^  ^  ^      ^   ^      ^
	//                                        44    50 53 56     63  67     74
```

I don't know why "ignored)" isn't getting an error as well.

## Expected

No errors

## versions

eslint 9.12.0 (about a year old)
svelte 5.46.0 (latest)
eslint-plugin-svelte 3.13.1 (latest)

**Comment by Stadly** [comment]
Agreed! This is very annoying, as it prevents me from giving a reason for ignoring stuff.

<img width="1032" height="122" alt="Image" src="https://github.com/user-attachments/assets/352ac630-1692-41d3-8714-0aad0cf67ca7" />

**Comment by ota-meshi** [comment]
Currently, this plugin's check rule cannot determine whether something is an Svelte compiler error code or a description, so the rule cannot reliably check for it.
If you're having trouble with false positives, please turn off the rule for now.
If you know of a better way to determine this, please share your ideas.

**Comment by marekdedic** [comment]
Drive-by comment: Do we need to know? What does Svelte interpret as a "commentary"? If it's just things in parentheses, that would be trivial to ignore... If it's anything that is not an error code, than that is hard (it would require keeping a list of valid codes, which is a pain to maintain)

**Comment by Stadly** [comment]
I gave it a go: #1484
